src/views/product-tutorials-view/components/product-view-content/types.ts

```diff
@@ -10,8 +10,7 @@ import {
   TutorialsStackProps,
 } from './components'
 
-interface FeaturedStackBlock
-  extends Pick<FeaturedStackProps, 'heading' | 'subheading'> {
+interface FeaturedStackBlock extends Omit<FeaturedStackProps, 'children'> {
```

Comment:
I think `Omit<>` might make more sense here, especially with `headingSlug` in place. I think I'd initially done `Pick<>` since I'd just come across it in my lil TypeScript journey 😅

Reply:
Love it 💯 Agreed it's definitely cleaner when you only have one thing to remove.

Reply:
Sometimes it depends though and its easier to read `Pick`, since you can scan what is being extended. Just something to think about.

Reply:
ah yeah great point - now I'm on the fence again 😝 might re-think this again as I work through the implementation of individual components 😅